src/Microsoft.Health.Fhir.Api.UnitTests/Features/Headers/FhirHeadersTests.cs

```diff
@@ -35,15 +37,16 @@ public FhirHeadersTests()
                     VersionId = version,
                     LastUpdated = date,
                 },
-            };
+            }.ToResourceElement();
         }
 
         [Fact]
         public void WhenSettingALocationHeader_ThenFhirResultHasALocationHeader()
         {
-            var locationUrl = new Uri($"http://localhost/{_mockResource.GetType().Name}/{_mockResource.Id}/_history/{_mockResource.Meta.VersionId}");
+            var locationUrl = new Uri($"http://localhost/{_mockResource.GetType().Name}/{_mockResource.Id}/_history/{_mockResource.VersionId}");
```

Comment:
nit: I don't think it matters since we are mocking it but for clarity purposes I think the URL is supposed to be something like `http://localhost/Observation/xxx` but it wil lbe `http://localhost/ResourceElement/xxx` after the refactor.

Reply:
Thanks! I'll need help trying to find things like this to tidy up